src/mentors.json

```diff
@@ -298,5 +298,30 @@
         "id": "_ArtOfCode"
       }
     ]
+  },
+  {
+    "id": "kamponistullar@gmail.com",
+    "name": "Ogundiji Bolade",
+    "avatar": "https://avatars1.githubusercontent.com/u/19638388?s=460&v=4",
+    "title": "Full Stack Web Developer",
+    "description": "Hi, I'm AdroitCode! Let me know if you need any help.",
+    "country": "Nigeria",
+    "tags": [
+      "php", "frontend", "javascript", "html", "css"
+    ],
+    "channels": [
+      {
+        "type": "email",
+        "id": "kamponistullar@gmail.com"
+      },
+      {
+        "type": "slack",
+        "id": "AdroitCode"
```

Comment:
Sorry for bother. Please follow this [post](https://medium.com/@moshfeu/how-to-find-my-member-id-in-slack-workspace-d4bba942e38c#88b8) to find the right memberID and you're ready to go!

Reply:
Never mind. I did it for you.